### ISSUE TITLE
Add original language to Translation request

### ISSUE THREAD

**Issue by gammaw** [EM-GUIDE/em-guide-tool#33]

**Comment by gammaw** [comment]
@revnandi sorry I forgot something here. Please also add a mandatory field for "Translated originally managed by:", which needs to be one of the mags.

**Comment by revnandi** [comment]
@gammaw And this field should be required if the status is "done"?

**Comment by gammaw** [comment]
@revnandi no, it's better to have it all times. So that the TL cannot be closed/saved without it. And the origin and target language should also be like this.
